lark/load_grammar.py

```diff
@@ -849,6 +846,32 @@ def _parse_grammar(text, name, start='start'):
     return PrepareGrammar().transform(tree)
 
 
+def _get_mangle(prefix, aliases, base_mangle=None):
```

Comment:
I would like this to be part of the public interface.

Reply:
It would be easy to bring back later. Tbh, I'm not sure I want it even for the internal interface :)

Reply:
Another approach: change names from string to tuple, and store the path there. Comparisons and checks should all work as usual, with minor changes. Then in the end of the process, all names will pass through a "mangler" which by default just joins the tuple with `__`, but can be provided to the class by the user.

That should work, right?